### ISSUE TITLE
Dataset and structure information

### ISSUE THREAD

**Issue by buaa-luzhi** [kilsen512/R3DCNN-tensorflow#7]
@kilsenp
Hello, So sorry to trouble you again. I really want to know the organization of the data set. Because the config.py script contains two gesture categories, "No gesture" and "blank". However, these two types of data are not present in the NV gesture data.
Can you show me the structure of the data in detail.
Thank you very much!

**Comment by kilianyp** [comment]
Hi,
no worries.
I was using this project for practical applications, where I needed a no gesture class.
I remember the blank category to be quite confusing to me, but from my understanding now it is necessary and cannot replace the no gesture class.
I can recommend this article: https://distill.pub/2017/ctc/ if you want to know more.

Unfortunately, I **never** properly tested it on the NV gesture data set which I certainly should have done to ensure the correctness of my implementation.

Have you checked out this repo: https://github.com/breadbread1984/R3DCNN
It seems to have been implemented closer with the NV dataset in mind.

**Comment by buaa-luzhi** [comment]
Firstly, I am so glad that you can help me again. 
For the past two years, I have also been doing *isolated* gesture recognition.  I've also faced the problems with no gesture class.  Later, we collected a lot of video files that did not contain hands to represent the no gesture class.  Therefore, the total output of the softmax classifier is equal to the 'predefined class' + 'no gesture class'.
I'd like to know what the predefined and non gesture video files of training network look like in your project.  I am eager for you to send two examples of video files to my email,   [by1603117@buaa.edu.cn](url)
Thanks so much for your help.
Besides,  for https://github.com/breadbread1984/R3DCNN, I have trained many times. I don't know why every time I test the local video using the pre-training model the output is random.
![70779146-ea4b0c00-1dbe-11ea-93cd-f2d4017d0816](https://user-images.githubusercontent.com/28650172/70957439-131a1c80-20b1-11ea-9737-3dbc0c960ca8.png)
The picture above shows the test results of one experiment. In addition, the code is too encapsulated to be easily modified.
Therefore, thanks for your sharing.

**Comment by buaa-luzhi** [comment]
Besides, in addition, is the 'no gesture' class required by the project or set by using CTC.  As far as I know, the concept of 'no gesture' class is introduced in the original text for the purpose of using CTC, and the actual training data is not set with 'no gesture' class alone.
Thanks again.  I'd really like to ask you more questions. Please forgive me.
